apps/snippets/scale.sh

```diff
@@ -5,6 +5,4 @@ check_meao_env
 
 deis limits:set web=300M/600M -a snippets-prod
 deis limits:set web=250m/1000m --cpu -a snippets-prod
-deis autoscale:set web --min=5 --max=15 --cpu-percent=80 -a snippets-prod
-
-deis scale clock=1 -a snippets-prod
+deis autoscale:set web --min=10 --max=30 --cpu-percent=80 -a snippets-prod
```

Comment:
Changed this to `min=10` from `min=15` which reflects what's on the servers now after recent optimization changes.